node/module.ts

```diff
@@ -200,9 +206,10 @@ class Module {
   static _cache: { [key: string]: Module } = Object.create(null);
   static _pathCache = Object.create(null);
   static globalPaths: string[] = [];
-  // Proxy related code removed.
   static wrapper = [
-    "(function (exports, require, module, __filename, __dirname) { ",
+    // We provide non standard timer APIs in the CommonJS wrapper
+    // to avoid exposing them in global namespace.
+    "(function (exports, require, module, __filename, __dirname, setTimeout, clearTimeout, setInterval, clearInterval) { ",
```

Comment:
I hope this doesn't break any third-party code...

aside: in the future we should use `v8::script_compiler::compile_function_in_context()` but that's not currently exposed by deno.

Reply:
We'll handle that when rewriting all of this in Rust